docs/GraphQL.md

```diff
@@ -601,7 +601,7 @@ packages and sources:
 type HasSBOM {
   id: ID!
   "SBOM subject"
-  subject: PackageOrSource!
+  subject: PackageOrArtifact!
```

Comment:
You also need to change on line 598 in text, but the rest looks good

Reply:
done.